src/background.ts

```diff
@@ -44,6 +44,7 @@ protocol.registerSchemesAsPrivileged([
 const store = new Store<{
   useGpu: boolean;
   savingSetting: SavingSetting;
+  hotkeySettings: HotkeySetting[];
```

Comment:
`hotkeySettings: Record<HotkeyAction, HotkeyCombo>`にするのはどうでしょう。
`for`で`HotkeyAction`を検索するコードが数箇所にあるので、楽になると思います。

---

How about making it `hotkeySettings: Record<HotkeyAction, HotkeyCombo>`?
There are a few places where the code to find `HotkeyAction` with `for`, so it would be easier.

Reply:
Record is an object so I'll have to define every hotkey not only in defaults but also in schema of the electron store. I think keeping two copies with this amount will be hard to maintain...

---

Recordはobjectですから、defaultsだけではなく、schemaにもショートカットずつ書かなければなりません。この数のコードの2つのコピーを保存するのはメンテナンス的に悪いと思います。